plotly/figure_factory/_facet_grid.py

```diff
@@ -25,6 +25,7 @@
 THRES_FOR_FLIPPED_FACET_TITLES = 10
 GRID_WIDTH = 1
 
+VALID_TRACE_TYPES = ['scatter', 'scattergl', 'histogram']
```

Comment:
I remember not liking the look of box or bar - it was very squashed up, but perhaps it was the dataset